src/lib/edge-functions/registry.mjs

```diff
@@ -295,7 +295,13 @@ export class EdgeFunctionsRegistry {
       ...route,
       pattern: new RegExp(route.pattern),
     }))
-    const functionNames = routes.filter(({ pattern }) => pattern.test(urlPath)).map((route) => route.function)
+    const functionNames = routes
+      .filter(({ pattern }) => pattern.test(urlPath))
+      .filter(
+        ({ function: name }) =>
+          !manifest.function_config[name]?.excluded_patterns.some((pattern) => new RegExp(pattern).test(urlPath)),
```

Comment:
I find this a bit hard to read. Do you think we could create an intermediate variable here, like `isExcludedPattern`?

Reply:
agreed, done in https://github.com/netlify/cli/pull/5366/commits/159c92c5f84cd7251d61aa6a9db2961d2b0206b6